src/ipc/ipc4/logging.c

```diff
@@ -78,7 +85,8 @@ static void mtrace_log_hook(size_t written, size_t space_left)
 
 	delta = k_uptime_get() - mtrace_notify_last_sent;
 
-	if (delta < IPC4_MTRACE_NOTIFY_MIN_DELTA_MS)
+	if (delta < IPC4_MTRACE_NOTIFY_MIN_DELTA_MS &&
+	    space_left > NOTIFY_BUFFER_CRIT_THRESHOLD)
```

Comment:
ipc message cost more cycles? disble ipc message also cause log missing?

Reply:
@btian1 Sending the IPC message is actually not that costly.